pkg/memio/mmap_linux.go

```diff
@@ -22,33 +22,43 @@ type syscalls interface {
 
 type calls struct{}
 
-func (c *calls) Mmap(fd int, page int64, mapSize int, prot int, callid int) ([]byte, error) {
-	return syscall.Mmap(fd, page, mapSize, prot, callid)
+func (c *calls) Mmap(fd int, page int64, mapSize int, prot int, flags int) ([]byte, error) {
+	return syscall.Mmap(fd, page, mapSize, prot, flags)
 }
 
 func (c *calls) Munmap(mem []byte) error {
 	return syscall.Munmap(mem)
 }
 
+func MMapPrivate(f *os.File) (data []byte, err error) {
```

Comment:
if you have this function, why did you write your own mmap call above?

Reply:
Good catch! left over... I'll revert this. Eventually, I found defining writing a simple mmap seems easiest.

Wanted to re-use as much as existing pkg, so tried to modify make modification here. Here seems mainly messing w/ memory w/ `/dev/mem`, and mmap is just the technique used, does not seems a general mmap pkg, and options like flags are fixed.  And I still don't understand why we need defining `syscalls` interface here, seems unnecessary, could be done by just a `unix.Mmap`. But I could be wrong, so best not messing w/ it in this PR.